packages/core/src/controllers/ApiController.ts

```diff
@@ -138,6 +138,16 @@ export const ApiController = {
     }
   },
 
+  getChains() {
+    const { caipNetwork } = NetworkController.state
```

Comment:
Let's separate the preperation-execution-conclusion parts of the function body with a new line. This will help us to keep them in a good pattern to make them more readable.

```suggestion
    const { caipNetwork } = NetworkController.state

```